src/libs/actions/Policy.js

```diff
@@ -691,15 +691,16 @@ function capitalizeFirstLetter(value) {
     return value.charAt(0).toUpperCase() + value.slice(1);
 }
 
+function getAllPolicies() {
+    return allPolicies;
+}
+
 /**
  * Generate a policy name based on an email and policy list.
  * @param {String} email
- * @param {Array} policyList
  * @returns {String}
  */
-function generateDefaultWorkspaceName(email, policyList) {
-    const policyListLength = policyList.length;
-    const hasPolicies = policyListLength > 0;
+function generateDefaultWorkspaceName(email) {
```

Comment:
You should be able to just use sessionEmail instead of passing an email parameter in